misc/config_tools/configurator/src/config.js

```diff
@@ -0,0 +1,7 @@
+
+export const ACRNConfig = {
+    defaultWorkingFolder: "/acrn-work/MyConfiguration/",

```

Comment:
Is that relative to the user's home folder?

Reply:
This file is deprecated, will be removed in the next commit.